src/types/IReverseButton.ts

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
You created one file for one API request, now you create one file for one type. don't you see the addiction?

Reply:
Now see